integration/fuse/src/main/java/alluxio/fuse/AlluxioJniFuseFileSystem.java

```diff
@@ -676,12 +698,15 @@ public void umount(boolean force) throws FuseException {
       } catch (InterruptedException e) {
         LOG.error("Unmount {} interrupted", mountpoint);
         Thread.currentThread().interrupt();
+        throw new AlluxioRuntimeException(Status.CANCELLED, "Unmount interrupted",
```

Comment:
nit: we can create the corresponding CancelledRuntimeException if it's not there.